### PR TITLE
Onboarding v2 & Misc. Updates

### PR DESCRIPTION
**Pull request by 0aveRyan**
## Proposed changes

This PR mainly focuses on Onboarding, however due to Cypress tests, some broken features and exploratory work last night a few other things slipped in when I was trying to cherry pick changes.

* New `/home/onboarding` UI & Tests _(Please test at desktop and mobile breakpoints!)_
* FIXED: Mobile Menu not working (and tests)
* FIXED: Brand colors not 1:1 with Bluehost brand guidelines (which forced some a11y updates in Staging/Help)
* ENHANCE: Make Product Card Images/Titles clickable
* ENHANCE: Shim Blue Sky as a Product Card in Services

## Type of Change

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Review by circlecube — APPROVED** [review]
I found a nit or two but nothing blocking. Overall, great job @0aveRyan!!

**Comment by wpscholar** [comment]
@0aveRyan I was greeted by the "plugin could not be loaded" screen several times before I saw the onboarding screen. Also, shouldn't I be seeing default content and tours? Clicking the buttons for each page just takes me to the "Add New" page screen.

**Comment by aaroncampbell** [comment]
> I was greeted by the "plugin could not be loaded" screen several times before I saw the onboarding screen.

This likely needs to be hunted down and fixed.

> Also, shouldn't I be seeing default content and tours? Clicking the buttons for each page just takes me to the "Add New" page screen.

The default content and tours are in the [add/editor-tours branch](/bluehost/bluehost-wordpress-plugin/tree/add/editor-tours) which will be a separate PR, but this one needs to merge first.

**Review by wpscholar — APPROVED** [review]

File: inc/admin/class-mods.php [hunk 1]
```diff
@@ -36,5 +36,30 @@ public static function return_instance() {
 	 *
 	 */
 	protected function primary_init() {
+		if ( isset( $_GET['page'] ) && 'bluehost' === filter_input( INPUT_GET, 'page', FILTER_SANITIZE_STRING ) ) {
+			\add_filter( 'admin_footer_text', array( $this, 'add_brand_to_admin_footer' ) );
+			// \add_filter( 'update_footer', array( $this, 'add_plugin_version_to_update_footer'), 50 );
```
[the remark this answers is not in the document]
Comment (0aveRyan) [reply inc/admin/class-mods.php:41]: yep, first filter is ready for primetime, second one needs iterating and discussion.

**Pull request merged**